scripts/bb_regression.py

```diff
@@ -24,7 +24,7 @@
 from top.model.bbox_3d import BoundingBoxRegressionModel
 from top.model.loss_util import orientation_loss, generate_bins
 
-from top.data.objectron_dataset_detection import Objectron
+from top.data.objectron_detection import Objectron
```

Comment:
changes in this file except for this are autoformatting artifacts, feel free to ignore.